### PR TITLE
benchmark workflow -> base R

### PR DESCRIPTION
**Pull request by vincentarelbundock**

### CODE REVIEW

**Comment by vincentarelbundock** [comment]
@etiennebacher I just re-wrote the benchmark script in mostly base R. No action need. Just a FYI.

File: .github/scripts/benchmarks.R [hunk 1]
```diff
@@ -17,6 +16,7 @@ out <- cross::run(
 
     bench::press(
       N = 75000,
+      N = 20,
```
Comment (etiennebacher) [inline .github/scripts/benchmarks.R:19]: This was just for testing I guess

**Comment by etiennebacher** [comment]
Thanks, I had tried with data.table but unnesting was surprisingly unfriendly compared to tidyverse.

File: .github/scripts/benchmarks.R (hunk 1 above)
Comment (vincentarelbundock) [reply .github/scripts/benchmarks.R:19]: oops

**Comment by vincentarelbundock** [comment]
Yeah, I think the pattern in `data.table` is a bit unintuitive, but using `by=` and `[[1]]` seems to work:

``` r
library(data.table)
url <- "https://vincentarelbundock.github.io/Rdatasets/csv/HistData/Guerry.csv"
dat <- na.omit(fread(url))

nested = dat[, list(data=list(.SD)), by=Region]

unnested = nested[, data[[1]], by = Region]

head(nested, 2)
>    Region                data
>    <char>              <list>
> 1:      E <data.table[17x23]>
> 2:      N <data.table[17x23]>

head(unnested, 2)
>    Region rownames  dept   Department Crime_pers Crime_prop Literacy Donations
>    <char>    <int> <int>       <char>      <int>      <int>    <int>     <int>
> 1:      E        1     1          Ain      28870      15890       37      5098
> 2:      E        4     4 Basses-Alpes      12935       7289       46      2733
>    Infants Suicides MainCity Wealth Commerce Clergy Crime_parents Infanticide
>      <int>    <int>   <char>  <int>    <int>  <int>         <int>       <int>
> 1:   33120    35039    2:Med     73       58     11            71          60
> 2:   23018    14238     1:Sm     76       49      5            70          12
>    Donation_clergy Lottery Desertion Instruction Prostitutes Distance  Area
>              <int>   <int>     <int>       <int>       <int>    <num> <int>
> 1:              69      41        55          46          13  218.372  5762
> 2:              37      80        32          29           2  351.399  6925
>    Pop1831
>      <num>
> 1:  346.03
> 2:  155.90
```

**Pull request merged**